bind.sh

```diff
@@ -133,15 +154,26 @@ fi
 if [[ -n "${cpus+x}" || -n "${mems+x}" ]]; then
   if command -v numactl &> /dev/null; then
       if [[ -n "${cpus+x}" ]]; then
-          set -- --physcpubind "${cpus[$rank]}" "$@"
+          set -- --physcpubind "${cpus[$local_rank]}" "$@"
       fi
       if [[ -n "${mems+x}" ]]; then
-          set -- --membind "${mems[$rank]}" "$@"
+          set -- --membind "${mems[$local_rank]}" "$@"
       fi
       set -- numactl "$@"
   else
       echo "Warning: numactl is not available, cannot bind to cores or memories" 1>&2
   fi
 fi
 
+# arguments may contain the substring %%LEGATE_GLOBAL_RANK%% which needs to be
+# be replaced with the actual computed rank for downstream processes to use
+count=0
```

Comment:
What is this for?

Reply:
cruft :) first attempt tried to `set` individual args with slices

Reply:
https://github.com/nv-legate/legate.core/pull/471/commits/b71a55698687ed76659682f3d104446af5d8dc64